curvefs/proto/mds.proto

```diff
@@ -167,4 +177,7 @@ service MdsService {
     rpc DeleteFs(DeleteFsRequest) returns (DeleteFsResponse);
     rpc AllocateS3Chunk(AllocateS3ChunkRequest) returns (AllocateS3ChunkResponse);
     rpc ListClusterFsInfo (ListClusterFsInfoRequest) returns (ListClusterFsInfoResponse);
+
+    // client lease
+    rpc RefreshSession(RefreshSessionRequest) returns (RefreshSessionResponse);
```

Comment:
Add this rpc interface in TopologyService maybe better, can deal request by topology_manager directly instead of pass by fs_manager.

Reply:
> Add this rpc interface in TopologyService maybe better, can deal request by topology_manager directly instead of pass by fs_manager.

agree!

Reply:
I've considered between mds and topology before.The information updated by Refreshsession currently is the txid of the partition, but it has at least one automatic umount responsibility, and there may be others later, so it is added to the unified entry.